### PR TITLE
feat(DMVP): AWS kafka module

### PR DESCRIPTION
**Pull request by aghamyan44**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# terraform-aws-msk
```
Comment (mrdntgrn) [inline README.md:1]: please have some minimal description what this module do and an maybe dummy code

File: variables.tf [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  description = "Determines whether to create SASL/SCRAM secret association"
+  type        = bool
+  default     = true
+}
+
+variable "kafka_auth" {
+  type = any
+  default = {
+    username = "dev",
+    password = "AAaa123456789!!"
```
Comment (aramkarapetian) [inline variables.tf]: Please remove default value

File: variables.tf [hunk 3]
```diff
@@ -0,0 +1,72 @@
+variable "name" {
+  type        = string
+  description = "Kafka name"
+  default     = "msk"
```
Comment (aramkarapetian) [inline variables.tf]: Please remove default value from name, this is security issue

File: variables.tf [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+variable "broker_node_instance_type" {
+  type        = string
+  default     = "kafka.t3.small"
+  description = "A list of the security groups to associate with the elastic network interfaces to control who can communicate with the cluster"
+}
+
+variable "number_of_broker_nodes" {
+  type        = number
+  default     = 3
```
Comment (aramkarapetian) [inline variables.tf]: Is there a way to reduce this to 2 or even one?

File: variables.tf [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+}
+
+variable "kms_key_owner_username" {
+  type        = string
+  description = "KMS key owner aws account username"
+}
+
+variable "ingress_access" {
+  type        = list(string)
+  default     = ["0.0.0.0/0"]
```
Comment (aramkarapetian) [inline variables.tf]: I think this one is also security problem, can we remove default value?

**Review by aramkarapetian — APPROVED** [review]
LGTM

**Pull request merged**